images/cilium/Dockerfile

```diff
@@ -3,8 +3,12 @@
 # Copyright 2020-2021 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-ARG CILIUM_BUILDER_IMAGE=docker.io/cilium/cilium-builder-dev:2be3763997e9439d3711f80741944b92f0c28ea9
-ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:e852cc7caf2d7a24102c05786f12b96eb106a2a0
+ARG CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:7c479d7c512c2075bbf72be10a2eb1362aa73176@sha256:e5835ca8779f9dc190b0e188de8066e3849c005bbe9ee989f94a9b83c8f8bd8f
+ARG CILIUM_RUNTIME_IMAGE=quay.io/cilium/cilium-runtime:397e2ee25f0dc5386dfdf1ef0ddadd499c74cd39@sha256:1c3bbed15fcb8a99df24c077bf69d870f962e3c6d00ace15f81eef9a448a0945
+
+# cilium-envoy from github.com/cilium/proxy
+#
+FROM quay.io/cilium/cilium-envoy:3b70fad0b9514720f33db82841907821202c1f02@sha256:8cca16ce66a0960a207cbf518ee2e0d923ae2a49207b154cdc37c2d95f583180 as cilium-envoy
```

Comment:
I think that for v1.9 we need to make this change somewhere else, these dockerfiles are not used. If you search this dockerfile, there's no other mention of this `cilium-envoy` container so this line is a noop.

Also I don't think we can just bump the builder/runtime images to the master version on older branches.

Reply:
thanks, let me take another look

Reply:
oh right, it's in root directory Dockerfile.